### PR TITLE
Update from v3.7 to v4.0

### PR DESCRIPTION
**Pull request by marcoesters**
Needed for anaconda-linter

# Package Information

Home: https://github.com/bastikr/boolean.py
Documentation: https://github.com/bastikr/boolean.py
Change log: https://github.com/bastikr/boolean.py/compare/v3.7...v4.0

### CODE REVIEW

**Review by varlackc — COMMENTED** [review]
Since the package is using `pip` we probably will need to use the `pip check` in the test section.
We also need to have `setuptools` as a `host` dependency.

Also the package seems to be failing for `linux-64`

**Comment by marcoesters** [comment]
> Since the package is using `pip` we probably will need to use the `pip check` in the test section. We also need to have `setuptools` as a `host` dependency.
> 
> Also the package seems to be failing for `linux-64`

All checks passed earlier, but maybe Prefect dragged its heels again...

Could you explain why `setuptools` needs to be both in `build` and in `host`?

File: recipe/meta.yaml [hunk 1]
```diff
@@ -11,19 +11,24 @@ source:
 
 build:
   number: 0
-  noarch: python
+  skip: True # [py<36]
   script: "{{ PYTHON }} -m pip install . --no-deps -vv"
 
 requirements:
+  build:
```
Comment (skupr-anaconda) [inline recipe/meta.yaml]: The `build` is not needed for python packages if we do not use compilers or other build tools. `setuptools` should be in `host`

File: recipe/meta.yaml [hunk 2]
```diff
@@ -11,19 +11,24 @@ source:
...
+    - setuptools
   host:
     - pip
     - python
+    - setuptools
```
Comment (skupr-anaconda) [inline recipe/meta.yaml:21]: There is missing `wheel` in `host`

File: recipe/meta.yaml [hunk 3]
```diff
@@ -11,19 +11,24 @@ source:
...
   run:
     - python
 
 test:
   imports:
     - boolean
```
Comment (skupr-anaconda) [inline recipe/meta.yaml:30]: I think there is missing:
```
  requires:
    - pip
```

File: recipe/meta.yaml [hunk 4]
```diff
@@ -11,19 +11,24 @@ source:
...
+  commands:
+    - pip check
 
 about:
   home: https://github.com/bastikr/boolean.py
```
Comment (skupr-anaconda) [inline recipe/meta.yaml:35]: Missing `license_family: BSD`

**Comment by marcoesters** [comment]
Thank you, @skupr-anaconda - I think I addressed all your comments.

**Review by skupr-anaconda — COMMENTED** [review]
Do not forget to disable `abs.yaml` if you are going to merge the PR and upload artifacts to defaults

**Review by skupr-anaconda — APPROVED** [review]

**Review by varlackc — APPROVED** [review]
LGTM

**Pull request merged**